tests/utils/invalids.py

```diff
@@ -55,9 +55,11 @@
     {"rhc_config_state": "x"*12},
     {"cpu_model": "x"*101},
     {"number_of_cpus": "35465"},
+    {"number_of_cpus": 4294967296},
     {"number_of_sockets": "35465"},
     {"cores_per_socket": "35465"},
     {"system_memory_bytes": "35465"},
+    {"system_memory_bytes": 18446744073709551617},
```

Comment:
Same here but just for MTU
```suggestion
    {"system_memory_bytes": 18446744073709551617},
    {"mtu": 18446744073709551617},
```